application/messages/media.php

```diff
@@ -0,0 +1,7 @@
+<?php defined('SYSPATH') OR die('No direct script access.');
+
+return [
+    'range' => 'The file size should be less than :param3 KB',
```

Comment:
Can we convert KB to MB? I doubt most users want to see "The file size should be less than 10000KB"